contributors/guide/README.md

```diff
@@ -157,7 +157,19 @@ _Improvements needed_
 
 ## Open a Pull Request
 
-PR workflow is described [here](https://git.k8s.io/community/contributors/devel/pull-requests.md#the-testing-and-merge-workflow).
+Pull requests are often called simply "PR".  This jargon in itself is a hurdle to newcomers, but the open source pull request model of development is widely documented and searchable.   Kubernetes generally follows the standard [github pull request](https://help.github.com/articles/about-pull-requests/) process, but there is a layer of additional kubernetes specific (and sometimes SIG specific) differences.
+
+The first difference you'll see is that a bot will begin applying structured labels to your PR.
+
+The bot may also make some helpful suggestions for commands to run in your PR to facilitate review.  These `/command` options can be entered in comments to trigger auto-labeling and notifications.  The command reference is [here](https://git.k8s.io/test-infra/commands.md).
+
+Common new contributor PR issues are:
+
+* not having correctly signed the CLA ahead of your first PR (see [Sign the CLA](#sign-the-cla) section)
+* finding the right SIG or reviewer(s) for the PR (see [Code Review](#code-review) section) and following any SIG specific contributing guidelines
+* dealing with test cases which fail on your PR, unrelated to the changes you introduce (see [Test Flakes](http://velodrome.k8s.io/dashboard/db/bigquery-metrics?orgId=1))
+
```

Comment:
I like this content. I would prefer to see it in the pull-requests.md file. That file might need a rework as well. There is doubled information in there (signing the CLA). I think I want it to live in contributors/guide, not contributors/devel.

Reply:
Yeah I think we're getting to where the contrib guide wants to lean on stronger content in other docs.  A FAQ section in the pull-requests.md would be a good add.  Same with a FAQ for the main testing.md like we discussed last week.

Reply:
Making a note in umbrella ticket to go over pull-requests.md and also add the squash commits step.